modules/eco-resuming-node-from-maintenance-mode-with-cr.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-resuming-node-from-maintenance-mode-with-cr_{context}"]
+= Resuming a node from maintenance mode with the NodeMaintenance CR
+
+You can resume a node from maintenance mode that was initiated with a `NodeMaintenance` CR by deleting the `NodeMaintenance` CR.
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in to the cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+* When your node maintenance task is complete, delete the active `NodeMaintenance` CR:
++
+[source,terminal]
+----
+$ oc delete -f nodemaintenance-cr.yaml
+----
++
```

Comment:
Optionally:

or
oc delete nm <maintenance-example>

Reply:
I think it is better as it was.